### DISCUSSION TITLE
Sound Components

### DISCUSSION THREAD

**Discussion by Demyxa** [FakeFishGames/Barotrauma#6251] [Ideas and suggestions]
I suggest adding a Sound Component, that takes in a signal to play a single Note at that specific pitch, or even at a specific frequency. Would allow for some more customizable Alarms, Submarine Features and the good old music track being recreated!

The component could also have different types of sound it could play (similar to a Minecraft Noteblock)

**Comment by hubismal** [comment]
MIDI support for instruments (maybe using a simple FluidSynth implementation) would be really nice, like what was done with SS14.
